### PR TITLE
alias: various tweaks

### PR DESCRIPTION
**Pull request by waldyrious**
I got caught off-guard about an alias I set not being permanently saved, so I added a caveat to the main description.

While I was at it, I also tweaked the descriptions of some examples.

### CODE REVIEW

File: pages/common/alias.md [hunk 1]
```diff
@@ -10,14 +11,14 @@
...
 
 `alias -p`
 
-- Turn rm an interative command:
+- Turn rm into an interactive command:
 
 `alias {{rm}}="{{rm -i}}"`
 
-- Override la as ls -a:
+- Create a `la` shortcut for `ls -a`:
```
Comment (agnivade) [inline pages/common/alias.md]: Maybe we can reword this to "Create `la` as a shortcut for `ls -a`" ?

**Comment by agnivade** [comment]
:+1:

**Comment by ostera** [comment]
This one is also a built-in, such as `source`. It'd vary from shell to shell rather than from os to os.

Other than that, being pragmatic, LGTM.

**Comment by waldyrious** [comment]
If there are actual significant differences, I agree we should probably remove it, or at least stick to behavior that is common. But that's a separate discussion :)
